actor-server/actor-core/src/main/protobuf/dialog.proto

```diff
@@ -239,6 +239,16 @@ message DialogCommands {
 
     message WriteMessageSelfAck {}
 
+    message QuotedMessage {
```

Comment:
quoted message body should be determined inside DialogCommandHandlers, not in DialogExtension's api clients.